packages/office-addin-cli/package.json

```diff
@@ -33,7 +33,6 @@
     "@typescript-eslint/eslint-plugin": "^4.26.0",
     "concurrently": "^6.0.0",
     "eslint": "7.28.0",
-    "eslint-plugin-office-addins": "^1.0.0",
```

Comment:
What's the dependency chain that causes office-addin-cli to need to remove the plug-in?

Reply:
eslint-plugin-office-addins -> office-addin-usage-data -> office-addin-cli -> eslint-plugin-office-addins

Reply:
This is what gets logged when I run `npm run build`:
eslint-plugin-office-addins -> office-addin-usage-data -> office-addin-cli -> eslint-plugin-office-addins